.github/workflows/update_wiki.yml

```diff
@@ -1,49 +1,46 @@
-name: Sync Docs
+name: Sync Docs (Merge)
+
 on:
-  push:
-    paths:
-    - '**.asciidoc'
-    - '**.md'
-    - '**.adoc'
-    branches:
-    - master
-  workflow_dispatch:
+  pull_request:
+    types: [closed]
+
```

Comment:
Pushing to `master` is discouraged in gitflow. It's probably best to not trigger actions with direct pushes to main, as those changes wouldn't have been reviewed in a PR. Since pushing without a PR also doesn't allow you to add the `release` or `documentation` tags, I think it would be inconsistent to have those actions run